frontend/src/scenes/paths/NewPaths.js

```diff
@@ -344,7 +345,7 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
                                                         Average time{' '}
                                                     </span>
                                                     {humanFriendlyDuration(
-                                                        pathItemCard.targetLinks[0].average_conversion_time
+                                                        pathItemCard.targetLinks[0].average_conversion_time / 1000
```

Comment:
was this the wrong unit before?

Reply:
Correct. 6 hours average time on Paths makes no sense 😂 .